devdocs.el

```diff
@@ -255,6 +270,22 @@ This is an alist containing `entries' and `types'."
     ('?# (concat (devdocs--path-file base) path))
     (_ (concat (file-name-directory base) path))))
 
+(defun devdocs--render-pre-tag-with-fontification (dom)
```

Comment:
Let's call this `devdocs--shr-tag-pre`, since I might want to add some additional markup on pre blocks, maybe a left margin or background color.